subsystems/SimpleMotorBase.java

```diff
@@ -0,0 +1,30 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public abstract class SimpleMotorBase extends SubsystemBase {
+    protected SpeedControllerGroup motorGroup;
+    protected double throttle;
+
+    public SimpleMotorBase(SpeedController[] motors, double throttle) {
```

Comment:
Add some documentation here explaining what this is for, all the other methods are pretty self explainitory